README-QWIKLABS.md

```diff
@@ -36,23 +33,35 @@ pipeline. Provision a service account with all permissions needed to run this wh
 There is a single Kubernetes Engine cluster with 2 nodes. The `datadog-agent.yaml` manifest creates a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) that runs the Datadog agent on
 each node in the cluster. Alongside the Datadog agents are nginx processes that feed metrics to Datadog.
 
-## Prerequisites
+## Initial Setup
 
-### Run Demo in a Google Cloud Shell
+### Configure gcloud
 
-Click the button below to run the demo in a [Google Cloud Shell](https://cloud.google.com/shell/docs/).
-
-[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/gke-datadog-demo.git&amp;cloudshell_image=gcr.io/graphite-cloud-shell-images/terraform:latest&amp;cloudshell_tutorial=README.md)
-
-
-All the tools for the demo are installed. When using Cloud Shell execute the following
-command in order to setup gcloud cli. When executing this command please setup your region
+All the tools for the demo are installed. When using Cloud Shell execute the following command in order to setup gcloud cli. When executing this command please setup your region
 and zone.
 
 ```console
 gcloud init
 ```
 
+### Tools
+
+1. Apache Bench
+2. Terraform
+
+The specific versions used may not be absolutely required but if you run into issues this may help. Please note, Terraform has already been installed in Cloud console for you.
+
+#### Install Apache Bench
+
+For many users this won't be necessary as many operating systems have Apache
+Bench pre-installed. However it is contained within the `apache2-utils` package
+for Ubuntu/Debian users, and the `httpd-tools` package for CentOS/Redhat users.
+
+### Configure Authentication
+
+Use `gcloud auth login <your.account@example.com> --no-launch-browser` to get a link to log in your
```

Comment:
Can you tell me how a set of personal credentials are to be used inside qwicklabs?  If it's necessary, can we derive the account email from an env var somehow?  I can see folks authenticating with the incorrect account and not having feedback if that's correct or not as they proceed through the demo.